python/py-graph-tool/Portfile

```
@@ -70,9 +72,25 @@ if {${name} ne ${subport}} {
                     port:py${python.version}-numpy \
                     port:py${python.version}-scipy
 
+    depends_lib-append \
+                    port:py${python.version}-matplotlib
+
     # graph-tool relies on Boost.Python, so make sure it is installed.
     require_active_variants boost[boost::version_nodot] python${python.version}
 
+    # remove after this issue is fixed:
+    # https://git.skewed.de/count0/graph-tool/-/issues/761
+    post-patch {
+        fs-traverse f ${worksrcpath}/src {
+            if { [file isfile ${f}]
+                && [regexp ".(cc|hh)" [file extension ${f}]] } {
```

In a regular expression `.` means "any character". Did you mean literally a period? If so, escape it with a backslash. Also, did you mean filenames that *contain* `cc` or `hh` or did you mean filenames that *end* with that? If the latter, anchor the regexp to the end of the string with `$`:
```suggestion
                && [regexp "\.(cc|hh)$" [file extension ${f}]] } {
```

---

Thanks, I used braces: `{\.(cc|hh)$}`. Note that EOL anchor is unnecessary because that's file `file extension` does, but it's good to be explicit for clarity.